corehq/apps/fixtures/dbaccessors.py

```diff
@@ -19,14 +20,28 @@ def get_number_of_fixture_data_types_in_domain(domain):
 @quickcache(['domain'], timeout=30 * 60)
 def get_fixture_data_types_in_domain(domain):
     from corehq.apps.fixtures.models import FixtureDataType
-    return list(FixtureDataType.view(
-        'by_domain_doc_type_date/view',
-        endkey=[domain, 'FixtureDataType'],
-        startkey=[domain, 'FixtureDataType', {}],
-        reduce=False,
-        include_docs=True,
-        descending=True,
-    ))
+
+    def get_results():
+        return list(FixtureDataType.view(
+            'by_domain_doc_type_date/view',
+            endkey=[domain, 'FixtureDataType'],
+            startkey=[domain, 'FixtureDataType', {}],
+            reduce=False,
+            include_docs=True,
+            descending=True,
+        ))
+
+    # band-aid workaround for a weird couch issue
+    #   where a deleted doc pops up in results as a dict
+    #   https://sentry.io/dimagi/commcarehq/issues/702737433/events/38122183220/
+    results = get_results()
+    for doc in results:
+        if type(doc) == dict:
+            # hopefully retry fixes it
```

Comment:
not sure that this is the best approach, a second call could have the same issue. Maybe better to just filter out the bad docs

Reply:
Updated